crates/net/network/src/config.rs

```diff
@@ -107,6 +107,10 @@ pub struct NetworkConfigBuilder<C> {
     discovery_v4_builder: Discv4ConfigBuilder,
     /// All boot nodes to start network discovery with.
     boot_nodes: Vec<NodeRecord>,
+    /// Nodes we always want to connect to.
+    trusted_nodes: Vec<NodeRecord>,
```

Comment:
all of these should be HashSet actually.

Reply:
Changed